packages/labs/tsserver-plugin/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "incremental": true,
+    "composite": true,
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
+    "target": "es2022",
+    "lib": ["es2022"],
+    "module": "commonjs",
+    "rootDir": "./src",
+    "moduleResolution": "node",
```

Comment:
can this and `module` be `nodenext` since the `package.json` doesn't have a `type: "module"`?